api/src/wfl/executor.clj

```diff
@@ -307,16 +314,17 @@
   "Create new submission from new `source` snapshot if available,
   writing its workflows to `details` table.
   Update statuses for active or failed workflows in `details` table.
-  Return `executor`."
-  [source executor]
+  Return updated `workload`."
+  [{:keys [executor source] :as workload}]
```

Comment:
Nit: the prior ordering of `source executor` makes a bit more logical sense, even though it's not alphabetical.